kolibri/utils/tests/test_version.py

```diff
@@ -430,3 +430,43 @@ def test_truncate_version(self):
             "1.0.0",
             version.truncate_version("1.15.1", truncation_level=version.MAJOR_VERSION),
         )
+
+    def test_normalize_version_to_semver_dev(self):
+        self.assertEqual(
+            version.normalize_version_to_semver(
+                "0.15.0a5.dev0+git.682.g0be46de2",
+            ),
+            "0.15.0-a.5.dev0.git.682.g0be46de2",
+        )
+
+    def test_normalize_version_to_semver_tripartite(self):
+        self.assertEqual(
+            version.normalize_version_to_semver(
+                "0.15.0",
+            ),
+            "0.15.0-c",
+        )
+
+    def test_normalize_version_to_semver_bipartite(self):
+        self.assertEqual(
+            version.normalize_version_to_semver(
+                "1.10.0",
```

Comment:
Sorry I missed this when we were looking at the code together, this should be `1.10` for it to be testing a two part (bipartite) version.